nixos/modules/services/continuous-integration/hydra/default.nix

```
@@ -277,16 +275,21 @@ in
 
     environment.variables = hydraEnv;
 
-    nix.extraOptions = ''
-      keep-outputs = true
-      keep-derivations = true
-
-
-    '' + optionalString (versionOlder (getVersion config.nix.package.out) "2.4pre") ''
-      # The default (`true') slows Nix down a lot since the build farm
-      # has so many GC roots.
-      gc-check-reachability = false
-    '';
+    nix.settings = mkMerge [
+      {
+        keep-outputs = true;
+        keep-derivations = true;
+        nix.settings.trusted-users = [ "hydra-queue-runner" ];
```

```suggestion
        trusted-users = [ "hydra-queue-runner" ];
```

Although hydra is about to be removed iiuc.

---

thanks!

---

> hydra is about to be removed iiuc

How? (just an off-topic question)

---

There's talk about removing the nixUnstable it depends on in favor of using flakes to install hydra. Maybe someone will maintain a "stable" hydra instead, if there's even such a thing.